src/reducers/auth.reducer.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN, LOGOUT } from '../actions/action_types';
+
+const reducer = (state = {}, action) => {
+    switch (action.type) {
+    case LOGIN:
+        return Object.assign({}, state, { loggedUser: true });
+    case LOGOUT:
+        return Object.assign({}, state, { loggedUser: false });
```

Comment:
yeh, we stay with es6. I saw recently what problems can there be with Object Rest/Spread is still a proposal! https://github.com/tc39/proposal-object-rest-spread CC: @ulaulala

Reply:
Interesting, I haven't know about any bugs for that. @Tuhaj could you please elaborate more? If I understood correctly from this issue (https://github.com/tc39/proposal-object-rest-spread/issues/52) it uses Object.assign under the hood.

Reply:
@kgajowy No bugs, but a lot of bother in case of deploy on SOME platforms. We recently tried to deploy a node app on Azure, and guess what happened: transpilers needed. Luckly on Heroku it went without any problems.